README.md

```diff
@@ -36,7 +36,7 @@ Example `stack.yml` for `mybb`:
 ```yaml
 services:
   mybb:
-    image: mybb/mybb:latest
+    build: .
```

Comment:
I don't believe this is supported for `docker swarm`/`docker stack` (only Compose). If that's the case, the title likely needs changing.

_(of course, it's possible I've just not been keeping up to date and this is now supported...)_